packages/cache-inmemory/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "scripts": {
     "watch": "tsc --build --watch",
-    "prepare": "cross-env NODE_ENV=production yarn run build",
+    "prepublishOnly": "tsc --build",
```

Comment:
in the files we have lost the cross env and the production env var. don't we need that?